src/contracts/core/PaymentCoordinatorStorage.sol

```diff
@@ -0,0 +1,107 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "src/contracts/interfaces/IAVSDirectory.sol";
+import "src/contracts/interfaces/IStrategyManager.sol";
+import "src/contracts/interfaces/IDelegationManager.sol";
+import "src/contracts/interfaces/IEigenPodManager.sol";
+import "src/contracts/interfaces/IPaymentCoordinator.sol";
+
+/**
+ * @title Storage variables for the `PaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ * @notice This storage contract is separate from the logic to simplify the upgrade process.
+ */
+abstract contract PaymentCoordinatorStorage is IPaymentCoordinator {
+
+    /*******************************************************************************
+                               CONSTANTS AND IMMUTABLES 
+    *******************************************************************************/
+
+    /// @notice The EIP-712 typehash for the contract's domain
+    bytes32 public constant DOMAIN_TYPEHASH =
+        keccak256("EIP712Domain(string name,uint256 chainId,address verifyingContract)");
```

Comment:
I know these are constant in our other contracts, but I'd like to start moving away from public constants. They bloat bytecode size and aren't needed.

Reply:
https://github.com/Layr-Labs/eigenlayer-contracts/pull/514/commits/6f3d65e2dc454dcc0a3a57451e003455a69fa0fb